<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="refresh" content="0; url=https://new-destination-link.com">
    <title>Redirecting...</title>
</head>
<body>
    <p>If you are not redirected automatically, follow this <a href="https://new-destination-link.com">link</a>.</p>
</body>
</html>


<!-- <!DOCTYPE html><html lang="en-us" >


<head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  
  
  
    <meta name="generator" content="Wowchemy 5.5.0 for Hugo" />
  

  
  










  







  
  
  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
  

  
  
  
    
      
      <link rel="preload" as="style" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap">
      <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap" media="print" onload="this.media='all'">
    
  

  
  
  
  
  
    
    
    
  
  

  <meta name="author" content="Ye Zheng, Ph.D." />

  
  
  
    
  
  <meta name="description" content="Ye Zheng personal website." />

  
  <link rel="alternate" hreflang="en-us" href="https://yezhengSTAT.github.io/" />

  
  
  
    <meta name="theme-color" content="#707070" />
  

  
  
    
    <script src="/js/mathjax-config.js"></script>
  

  

  <link rel="stylesheet" href="/css/vendor-bundle.min.c7b8d9abd591ba2253ea42747e3ac3f5.css" media="print" onload="this.media='all'">

  
  
  
    
    
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha512-W0xM4mr6dEP9nREo7Z9z+9X70wytKvMGeDsj7ps2+xg5QPrEBXC8tAW1IFnzjR6eoJ90JmCnFzerQJTLzIEHjA==" crossorigin="anonymous" media="print" onload="this.media='all'">
    

    
    
    
    
      
      
      
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
        
      
    
    
    

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.2.1/build/styles/github.min.css" crossorigin="anonymous" title="hl-light" media="print" onload="this.media='all'">
          <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.2.1/build/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark" media="print" onload="this.media='all'" disabled>
        
      
    

    
    
    

    

    
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js" integrity="" crossorigin="anonymous" async></script>
      
    
      
      

      
      

      
    
      
      

      
      

      
    
  

  
  
  
  
  
  <link rel="stylesheet" href="/css/wowchemy.9aeea823e57bd46b2474b6d8edbba3ce.css" />

  



  


  


  




  
  
  
    <script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
  

  
    <link rel="alternate" href="/index.xml" type="application/rss+xml" title="" />
  

  
    <link rel="manifest" href="/manifest.webmanifest" />
  

  <link rel="icon" type="image/png" href="/media/icon_hu73f7157ab925ae64d73c94fd5ceef54d_49900_32x32_fill_lanczos_center_3.png" />
  <link rel="apple-touch-icon" type="image/png" href="/media/icon_hu73f7157ab925ae64d73c94fd5ceef54d_49900_180x180_fill_lanczos_center_3.png" />

  <link rel="canonical" href="https://yezhengSTAT.github.io/" />

  
  
  
  
  
  
  
  
    
    
  
  

  
  
    
    
  
  <meta property="twitter:card" content="summary" />
  
    <meta property="twitter:site" content="@yezhengSTAT" />
    <meta property="twitter:creator" content="@yezhengSTAT" />
  
  <meta property="og:site_name" content="" />
  <meta property="og:url" content="https://yezhengSTAT.github.io/" />
  <meta property="og:title" content="" />
  <meta property="og:description" content="Ye Zheng personal website." /><meta property="og:image" content="https://yezhengSTAT.github.io/media/icon_hu73f7157ab925ae64d73c94fd5ceef54d_49900_512x512_fill_lanczos_center_3.png" />
    <meta property="twitter:image" content="https://yezhengSTAT.github.io/media/icon_hu73f7157ab925ae64d73c94fd5ceef54d_49900_512x512_fill_lanczos_center_3.png" /><meta property="og:locale" content="en-us" />
  
    
      <meta property="og:updated_time" content="2030-06-01T13:00:00&#43;00:00" />
    
  

  

<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "WebSite",
  "potentialAction": {
    "@type": "SearchAction",
    "target": "https://yezhengSTAT.github.io/?q={search_term_string}",
    "query-input": "required name=search_term_string"
  },
  "url": "https://yezhengSTAT.github.io/"
}
</script>


  

  

  





  <title></title>
</head>


<body id="top" data-spy="scroll" data-offset="70" data-target="#navbar-main" class="page-wrapper   "  >

  
  
  
  
  
  
  
  
  
  <script src="/js/wowchemy-init.min.2ed908358299dd7ab553faae685c746c.js"></script>

  


<aside class="search-modal" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#" aria-label="Close"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search" class="form-control"
        aria-label="Search...">
        
      </div>

      
      

      

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>



  <div class="page-header">
    












<header class="header--fixed">
  <nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
    <div class="container-xl">

      
      <div class="d-none d-lg-inline-flex">
        <a class="navbar-brand" href="/"></a>
      </div>
      

      
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar-content" aria-controls="navbar-content" aria-expanded="false" aria-label="Toggle navigation">
      <span><i class="fas fa-bars"></i></span>
      </button>
      

      
      <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
        <a class="navbar-brand" href="/"></a>
      </div>
      

      
      
      <div class="navbar-collapse main-menu-item collapse justify-content-start" id="navbar-content">

        
        <ul class="navbar-nav d-md-inline-flex">
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#about" data-target="#about"><span>Home</span></a>
          </li>

          
          

          

          
          
          
            
              
            
          

          

          
          
          
          

          
            
              
              
            
            
          

          <li class="nav-item">
            <a class="nav-link " href="https://pages.stat.wisc.edu/~yezheng/CV_YeZheng_2022.pdf" target="_blank" rel="noopener"><span>CV</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#publications" data-target="#publications"><span>Publications</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#projects" data-target="#projects"><span>Research</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#software" data-target="#software"><span>Software</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#teaching" data-target="#teaching"><span>Teaching&Mentoring</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#activity" data-target="#activity"><span>Professional Activity</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#posts" data-target="#posts"><span>Blogs</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#contact" data-target="#contact"><span>Contact</span></a>
          </li>

          
          

        

          
        </ul>
      </div>

      <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">

        
        
          
        

        
        
        <li class="nav-item">
          <a class="nav-link js-search" href="#" aria-label="Search"><i class="fas fa-search" aria-hidden="true"></i></a>
        </li>
        

        
        
        
        <li class="nav-item dropdown theme-dropdown">
          <a href="#" class="nav-link" data-toggle="dropdown" aria-haspopup="true" aria-label="Display preferences">
            <i class="fas fa-moon" aria-hidden="true"></i>
          </a>
          <div class="dropdown-menu">
            <a href="#" class="dropdown-item js-set-theme-light">
              <span>Light</span>
            </a>
            <a href="#" class="dropdown-item js-set-theme-dark">
              <span>Dark</span>
            </a>
            <a href="#" class="dropdown-item js-set-theme-auto">
              <span>Automatic</span>
            </a>
          </div>
        </li>
        

        
        

      </ul>

    </div>
  </nav>
</header>


  </div>

  <div class="page-body">
    
    
    

    


  









  
<span class="js-widget-page d-none"></span>





  
  
  
  





  

  
  
  
  

  

  

  
  
  
  

  

  

  

  
  

  

  
  

  
  
  

  
  
  
  
  

  
  

  

  
  

  
  

  
  <section id="about" class="home-section wg-about  "  >
   <div class="home-section-bg " >
     
   </div>
    <div class="container">

    

      




  










<div class="row">
  <div class="col-12 col-lg-4">
    <div id="profile">

      
      
      <img class="avatar avatar-circle"
           width="270" height="270"
           src="/authors/admin/avatar_hued0274b592f478cb9c941f5cf33a457b_5784633_270x270_fill_q100_lanczos_center.jpg" alt="Ye Zheng, Ph.D.">
      

      <div class="portrait-title">

        <h2>Ye Zheng, Ph.D.</h2>

        <h3>Tenure-track Assistant Professor and NIH R00 Fellow</h3>

        
        <h3>
          <a href="https://www.mdanderson.org/research/departments-labs-institutes/departments-divisions/bioinformatics-and-computational-biology.html" target="_blank" rel="noopener">
          <span>Department of Bioinformatics and Computational Biology, University of Texas MD Anderson Cancer Center</span>
          </a>
        </h3>
        
      </div>

      <ul class="network-icon" aria-hidden="true">
        
        
        
        
        
        
        
        
          
        
        <li>
          <a href="https://www.stat.wisc.edu/~yezheng/CV_YeZheng.pdf" target="_blank" rel="noopener" aria-label="cv">
            <i class="ai ai-cv big-icon"></i>
          </a>
        </li>
        
        
        
        
        
        
        
        
          
        
        <li>
          <a href="https://scholar.google.com/citations?user=8VAZHgQAAAAJ&amp;hl=en" target="_blank" rel="noopener" aria-label="google-scholar">
            <i class="ai ai-google-scholar big-icon"></i>
          </a>
        </li>
        
        
        
        
          
        
        
        
        
        
          
        
        <li>
          <a href="https://github.com/yezhengSTAT" target="_blank" rel="noopener" aria-label="github">
            <i class="fab fa-github big-icon"></i>
          </a>
        </li>
        
        
        
        
        
        
        
        
          
        
        <li>
          <a href="https://www.protocols.io/view/cut-amp-tag-data-processing-and-analysis-tutorial-e6nvw93x7gmk/v1" target="_blank" rel="noopener" aria-label="protocols">
            <i class="ai ai-protocols big-icon"></i>
          </a>
        </li>
        
        
        
        
          
        
        
        
        
        
          
        
        <li>
          <a href="https://twitter.com/yezhengstat" target="_blank" rel="noopener" aria-label="twitter">
            <i class="fab fa-twitter big-icon"></i>
          </a>
        </li>
        
        
        
        
          
        
        
        
        
        
          
        
        <li>
          <a href="https://www.linkedin.com/in/ye-zheng-1a167782/" target="_blank" rel="noopener" aria-label="linkedin">
            <i class="fab fa-linkedin big-icon"></i>
          </a>
        </li>
        
      </ul>

    </div>
  </div>
  <div class="col-12 col-lg-8">

    
    <h1><font size="6">Biography</font></h1>

    <div class="article-style">
      <p><font size="4"> Ye Zheng is an <a href="https://reporter.nih.gov/search/FF400P9uy0OX2wY98azYyg/project-details/10572539" target="_blank" rel="noopener">NIH/NHGRI K99/R00 fellow</a> and a tenure-track Assistant Professor in the <a href="https://www.mdanderson.org/research/departments-labs-institutes/departments-divisions/bioinformatics-and-computational-biology.html" target="_blank" rel="noopener">Bioinformatics and Computational Biology Department of the University of Texas MD Anderson Cancer Center</a>. Dr. Zheng received her postdoctoral training at the Fred Hutchinson Cancer Center from both molecular biology and quantitative modelingperspectives mentored by <a href="https://research.fredhutch.org/henikoff/en.html" target="_blank" rel="noopener">Dr. Steven Henikoff</a> and <a href="https://scholar.google.com/citations?user=h9E70BEAAAAJ&amp;hl=en" target="_blank" rel="noopener">Dr. Raphael Gottardo</a>. She has also established close collaborations with <a href="https://research.fredhutch.org/turtle/en.html" target="_blank" rel="noopener">Dr. Cameron Turtle</a> and <a href="https://research.fredhutch.org/newell/en.html?_gl=1*22rx9b*_ga*ODQ0NDE4ODc2LjE1NzI2Mzk3Mzg.*_ga_CMZTF4L2MS*MTY1ODE4MzExNC4zLjEuMTY1ODE4MzcwNy4w&amp;_ga=2.37524896.1834883733.1658169553-844418876.1572639738&amp;_gac=1.84158059.1654895497.CjwKCAjw14uVBhBEEiwAaufYx33C2qa6-50yzpX_nFjMkBLtXLDOjibsR2KXay8qStxZLfMIlPmFvxoCet8QAvD_BwE" target="_blank" rel="noopener">Dr. Evan Newell</a> to decipher the CAR-T cell immunotherapy response variations. Before her postdoctoral training, Dr. Zheng received a Ph.D. in Statistics from the University of Wisconsin-Madison under the supervision of <a href="https://biostat.wiscweb.wisc.edu/staff/keles-sunduz/" target="_blank" rel="noopener">Dr. Sündüz Keleş</a>, and her dissertation topics were centered around statistical modelings of three-dimensional chromatin structure (3D genomics) for promoter-enhancer inference.</p>
<p>At MD Anderson Cancer Center, Dr. Zheng leads a quantitative research group dedicated to the statistical modeling and computational pipeline development using bulk and single-cell transcriptomics, proteomics, epigenomics, and 3D genomics data to address biological and clinical challenges. Her wet lab specializes in the epigenomic profiling of the Formalin-Fixed, Paraffin-Embedded (FFPE) samples.</p>
<p>The goal of Dr. Zheng&rsquo;s research group is to solve biological and clinically important, and methodologically challenging problems by innovating cutting-edge statistical models. <strong>The group is actively hiring at all levels, including but not limited to Postdoc, Ph.D., Master students, technician and undergraduate or graduate interns</strong>, and open to discussion and collaboration. This highly interdisciplinary group looks forward to being inspired and motivated by the novel and intriguing problems in other disciplines.</p>
<h2 id="hiring-projects">Hiring Projects</h2>
<ol>
<li><strong>[3D genomics] Investigating the three-dimensional chromatin organization and the long-range gene regulation</strong> through multimodality integrative modeling and accompanying software development, using data such as scHi-C and scRNA-seq.</li>
<li><strong>[Epigenomics] RNA PolII profiling on the formalin-fixed paraffin-embedded (FFPE) samples</strong> provide a cost-effective and robust approach to generate critical data for cancer research and motivate new association and prediction models with patient phenotypes. <strong>This project also has a wet lab training option and is actively hiring a technician.</strong></li>
<li><strong>[Epigenomics] Single-cell epigenomics data are known for their ultra-sparsity.</strong> Denoising and imputation models are needed to gain useful cell information and integrate across epigenomic markers.</li>
<li><strong>[Proteomics] Cell surface protein measurement</strong> can provide deeper and standardized single-cell cell-type annotations and status descriptions. The project integrates CITE-seq and Cytomery data across the study and platform for joint disease analysis.</li>
<li><strong>[Immunotherapy and Multi-omics] Statistical modeling and computational analysis of immunological and immunotherapeutic Studies</strong> using multi-omics bulk and single-cell genomics data to decipher key genotypic and phenotypic features that drive efficacy versus toxicity in <strong>CAR-T cell immunotherapy</strong>.</li>
<li><strong>[ML] Genomic-disease association machine learning modeling</strong> leverage the unprecedented resolution of single-cell data for unraveling genomic underpinnings of disease phenotypes and therapeutic responses.</li>
</ol>
<p>Please send your CV/resume, a brief cover letter describing your relevant experience and motivations, a GitHub link to the repository or any other materials that can best demonstrate your programming skills, and any related research manuscripts/writing samples (if applicable) to <a href="mailto:yzheng8@mdanderson.org">yzheng8@mdanderson.org</a>.</p>
</font> 

    </div>

    <div class="row">

      
      <div class="col-md-5">
        <div class="section-subheading">Interests</div>
        <ul class="ul-interests mb-0">
          
          <li>Statistical Genomics</li>
          
          <li>Computational Biology</li>
          
          <li>Multi-omics</li>
          
          <li>Immunotherapy and Cancer Study</li>
          
        </ul>
      </div>
      

      
      <div class="col-md-7">
        <div class="section-subheading">Education</div>
        <ul class="ul-edu fa-ul mb-0">
          
          <li>
            <i class="fa-li fas fa-graduation-cap"></i>
            <div class="description">
              <p class="course">Ph.D. in Statistics - Minor in Quantitative Biology, 2019</p>
              <p class="institution">University of Wisconsin - Madison</p>
            </div>
          </li>
          
          <li>
            <i class="fa-li fas fa-graduation-cap"></i>
            <div class="description">
              <p class="course">B.E. in Statistics, 2014</p>
              <p class="institution">Renmin University of China</p>
            </div>
          </li>
          
        </ul>
      </div>
      

    </div>
  </div>
</div>


    

    </div>
  </section>

  

  
  
  
  

  

  

  
  
  
  

  

  

  

  
  

  

  
  

  
  
  

  
  
  
  
  

  
  

  

  
  

  
  

  
  <section id="experience" class="home-section wg-experience  "  >
   <div class="home-section-bg " >
     
   </div>
    <div class="container">

    
      <div class="row  ">
      
        
          <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
            <h1 class="mb-0">Professional Experience</h1>
            
          </div>
        
      
    

      




<div class="col-12 col-lg-8">
  

  
  

  
  
  <div class="row experience">
    
    <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
        <div class="col ">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
        <span class="badge badge-pill border ">&nbsp;</span>
      </div>
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
    </div>
    
    <div class="col py-2">
      <div class="card">
        <div class="card-body">
          <div class="d-flex align-content-start">
            <div class="mr-2 mb-2"><a href="https://www.fredhutch.org/" target="_blank" rel="noopener"><img src="/media/icons/brands/fredhutch.svg" width="56px" height="56px" alt="Fred Hutchinson Cancer Center" loading="lazy"></a></div>
            <div>
          

              <div class="section-subheading card-title exp-title text-muted my-0">Postdoctoral Research Fellow</div>
              <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://www.fredhutch.org/" target="_blank" rel="noopener">Fred Hutchinson Cancer Center</a></div>
              <div class="text-muted exp-meta">
                Nov 2019 –
                
                  Sep 2024
                
                
                  <span class="middot-divider"></span>
                  <span>Seattle, WA, USA</span>
                
              </div>
            </div>
          </div>
          

          <div class="card-text"><p><strong>Integrative modeling of bulk and single-cell transcriptomics, epigenomics and proteomics</strong>:</p>
<ul>
<li>Developed FFPE epigenomic landscape processing and analysis pipeline. Proposed customized normal- ization method for hypertranscription tumor samples. - Developed normalization method, ADTnorm, for CITE-seq data to remove the technical batch effect and facilitate data integration across studies. - Constructed data processing and analysis pipeline for CUT&amp;RUN and CUT&amp;Tag data and created spike-in free normalization method for multiple sample comparison. - Developed statistical models and computational tools for integrative analysis of single-cell 3D genomics, transcriptomics and epigenomics for gene cis-regulatory mechanism discovery.</li>
</ul>
<p><strong>Wet lab experimental training</strong>:</p>
<ul>
<li>Learned and implemented epigenomic experiments, including FFPE-CUTAC, CUT&amp;Tag, scCUT&amp;Tag, scMulTI-Tag, to collect data for multiple ongoing projects in the Henikoff lab.</li>
</ul>
<p><strong>Chimeric antigen receptor T (CAR-T) Cell Immunotherapy</strong>:</p>
<ul>
<li>Multi-omics integrative analysis to profile the genomic signatures of CAR-T cell therapy products from transcriptomics and epigenomics perspectives using CITE-seq, CUT&amp;RUN and scCUT&amp;Tag data. - Constructed tree-based machine learning models to detect genomic features associated with CAR-T cell immunotherapy efficacy and toxicity.</li>
</ul>
<p><strong>Statistical consulting for genomics and biomedical studies</strong>:</p>
<ul>
<li>Statistical support for single-cell RNA-seq analysis in Hsieh Lab, trajectory analysis in Prlic Lab, and collaborative analysis of cytotoxic HDAC inhibition with Seattle Children’s Hospital.</li>
</ul>
</div>
        </div>
      </div>
    </div>
  </div>
  
  <div class="row experience">
    
    <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
        <span class="badge badge-pill border ">&nbsp;</span>
      </div>
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
    </div>
    
    <div class="col py-2">
      <div class="card">
        <div class="card-body">
          <div class="d-flex align-content-start">
            <div class="mr-2 mb-2"><a href="https://www.biostat.wisc.edu/" target="_blank" rel="noopener"><img src="/media/icons/brands/uwmadison.svg" width="56px" height="56px" alt="Department of Statistics, Department of Biostatistics &amp; Medical Informatics, University of Wisconsin - Madison" loading="lazy"></a></div>
            <div>
          

              <div class="section-subheading card-title exp-title text-muted my-0">Research Assistant</div>
              <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://www.biostat.wisc.edu/" target="_blank" rel="noopener">Department of Statistics, Department of Biostatistics &amp; Medical Informatics, University of Wisconsin - Madison</a></div>
              <div class="text-muted exp-meta">
                Sep 2014 –
                
                  Oct 2019
                
                
                  <span class="middot-divider"></span>
                  <span>Madison, WI, USA</span>
                
              </div>
            </div>
          </div>
          

          <div class="card-text"><p><strong>Dissertation Research</strong>:</p>
<ul>
<li>Developed biologically motivated hierarchical generative model to investigate 3D chromatin architectures using Hi-C data and investigated the genomic features involving repetitive regions of the genomes.</li>
<li>Developed a computational tool for fast simulation of 3D proximity ligation sequencing data.</li>
<li>Constructed hierarchical testing to detect differential 3D genome interactions with precise False Discovery Rate control.</li>
<li>Investigated protein-DNA interactions residing in repetitive regions and integrated multi-mapping reads into Encyclopedia of DNA Elements (ENCODE) ChIP-seq data processing pipeline.</li>
</ul>
<p><strong>Collaborative Work with the Bresnick Lab</strong>:</p>
<ul>
<li>Leveraged multi-omics analysis, particularly using ATAC-seq and RNA-seq data, to reveals GATA/Heme regulation mechanism in controlling hemoglobin synthesis and erythrocyte development.</li>
<li>Investigated the impact of single nucleotide mutation in the Ets motif of GATA2 enhancer on its function to control hematopoiesis through a comprehensive transcriptomic differential analysis.</li>
</ul>
</div>
        </div>
      </div>
    </div>
  </div>
  
  <div class="row experience">
    
    <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
        <span class="badge badge-pill border ">&nbsp;</span>
      </div>
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
    </div>
    
    <div class="col py-2">
      <div class="card">
        <div class="card-body">
          <div class="d-flex align-content-start">
            <div class="mr-2 mb-2"><a href="https://www.research.ibm.com/labs/china/" target="_blank" rel="noopener"><img src="/media/icons/brands/IBM.svg" width="56px" height="56px" alt="Business Analysis, IBM" loading="lazy"></a></div>
            <div>
          

              <div class="section-subheading card-title exp-title text-muted my-0">Data Scientist Intern</div>
              <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://www.research.ibm.com/labs/china/" target="_blank" rel="noopener">Business Analysis, IBM</a></div>
              <div class="text-muted exp-meta">
                Feb 2014 –
                
                  Jun 2014
                
                
                  <span class="middot-divider"></span>
                  <span>Beijing, China</span>
                
              </div>
            </div>
          </div>
          

          <div class="card-text"><p>Projects:</p>
<ul>
<li>Developed dynamic text mining model using IBM communication database to infer topic networks.</li>
<li>Constructed a modified Latent Dirichlet Allocation model to optimize the CPU usage of IBM servers.</li>
</ul>
</div>
        </div>
      </div>
    </div>
  </div>
  
  <div class="row experience">
    
    <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
        <span class="badge badge-pill border ">&nbsp;</span>
      </div>
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
    </div>
    
    <div class="col py-2">
      <div class="card">
        <div class="card-body">
          <div class="d-flex align-content-start">
            <div class="mr-2 mb-2"><a href="http://info.ruc.edu.cn/academic_professor.php?teacher_id=71" target="_blank" rel="noopener"><img src="/media/icons/brands/ruc.svg" width="56px" height="56px" alt="School of Information, Renmin University of China" loading="lazy"></a></div>
            <div>
          

              <div class="section-subheading card-title exp-title text-muted my-0">Project Assistant</div>
              <div class="section-subheading card-title exp-company text-muted my-0"><a href="http://info.ruc.edu.cn/academic_professor.php?teacher_id=71" target="_blank" rel="noopener">School of Information, Renmin University of China</a></div>
              <div class="text-muted exp-meta">
                Feb 2012 –
                
                  Jan 2014
                
                
                  <span class="middot-divider"></span>
                  <span>Beijing, China</span>
                
              </div>
            </div>
          </div>
          

          <div class="card-text">Developed multi-objectives operations research model to improve proposals grouping accuracy and efficiency utilizing Multi-Objective Particle Swarm Optimization (MOPSO) algorithm for optimization.</div>
        </div>
      </div>
    </div>
  </div>
  
  <div class="row experience">
    
    <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
        <span class="badge badge-pill border ">&nbsp;</span>
      </div>
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
    </div>
    
    <div class="col py-2">
      <div class="card">
        <div class="card-body">
          <div class="d-flex align-content-start">
            <div class="mr-2 mb-2"><a href="http://www.bioinformatics.uottawa.ca/stephane/index.html" target="_blank" rel="noopener"><img src="/media/icons/brands/uottawa.svg" width="56px" height="56px" alt="Department of Biology, Mathematics and Statistics, University of Ottawa" loading="lazy"></a></div>
            <div>
          

              <div class="section-subheading card-title exp-title text-muted my-0">Research Assistant</div>
              <div class="section-subheading card-title exp-company text-muted my-0"><a href="http://www.bioinformatics.uottawa.ca/stephane/index.html" target="_blank" rel="noopener">Department of Biology, Mathematics and Statistics, University of Ottawa</a></div>
              <div class="text-muted exp-meta">
                Jun 2013 –
                
                  Sep 2013
                
                
                  <span class="middot-divider"></span>
                  <span>Ottawa, Canada</span>
                
              </div>
            </div>
          </div>
          

          <div class="card-text">Investigated Approximate Bayesian Computation (ABC), ABC–Markov Chain Monte Carlo and ABC– Sequential Monte Carlo samplers in estimating the transmission networks of viruses in human populations.</div>
        </div>
      </div>
    </div>
  </div>
  
  <div class="row experience">
    
    <div class="col-auto text-center flex-column d-none d-sm-flex">
      <div class="row h-50">
        <div class="col border-right">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
      <div class="m-2">
        <span class="badge badge-pill border ">&nbsp;</span>
      </div>
      <div class="row h-50">
        <div class="col ">&nbsp;</div>
        <div class="col">&nbsp;</div>
      </div>
    </div>
    
    <div class="col py-2">
      <div class="card">
        <div class="card-body">
          <div class="d-flex align-content-start">
            <div class="mr-2 mb-2"><a href="https://www.hku.hk/" target="_blank" rel="noopener"><img src="/media/icons/brands/hku.svg" width="56px" height="56px" alt="Department of Statistics and Actuarial Science, The University of Hong Kong" loading="lazy"></a></div>
            <div>
          

              <div class="section-subheading card-title exp-title text-muted my-0">Exchange Study</div>
              <div class="section-subheading card-title exp-company text-muted my-0"><a href="https://www.hku.hk/" target="_blank" rel="noopener">Department of Statistics and Actuarial Science, The University of Hong Kong</a></div>
              <div class="text-muted exp-meta">
                Sep 2012 –
                
                  Jan 2013
                
                
                  <span class="middot-divider"></span>
                  <span>Hong Kong, China</span>
                
              </div>
            </div>
          </div>
          

          
        </div>
      </div>
    </div>
  </div>
  
  
</div>


    
      </div>
    

    </div>
  </section>

  

  
  
  
  

  

  

  
  
  
  

  

  

  

  
  

  

  
  

  
  
  

  
  
  
  
  

  
  

  

  
  

  
  

  
  <section id="publications" class="home-section wg-pages  "  >
   <div class="home-section-bg " >
     
   </div>
    <div class="container">

    
      <div class="row  ">
      
        
          <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
            <h1 class="mb-0">Publications</h1>
            
          </div>
        
      
    

      









  









  
  
  
  
  



















  




<div class="col-12 col-lg-8">

  <p>+: co-first authors, ++: co-corresponding authors</p>
<h3 id="epigenomics-and-ffpe">Epigenomics and FFPE</h3>
<ol>
<li>
<p>Henikoff S+, <strong>Zheng Y+</strong>, Paranal R, Xu Y, Greene J, Henikoff J, Russell Z, Szulzewsky F, Thirimanne H, Kugel S, Holland E, Ahmad K. RNA Polymerase II at histone genes predicts outcome in human cancer. Under review of Science. (2024) Previous version is available at: <a href="https://www.biorxiv.org/content/10.1101/2024.02.28.582647v3" target="_blank" rel="noopener">https://www.biorxiv.org/content/10.1101/2024.02.28.582647v3</a>.</p>
</li>
<li>
<p>Henikoff S, <strong>Zheng Y</strong>, Ahmad K. Mitotic errors do not explain aneuploidy in cancer. Under review of Trends in Genetics. (2024)</p>
</li>
<li>
<p>[Book] Savonen et al. Choosing Genomics Tools. (2023) Chapter 19 CUT&amp;RUN and CUT&amp;Tag. <a href="https://hutchdatascience.org/Choosing_Genomics_Tools/about-the-authors.html" target="_blank" rel="noopener">Full author list</a>. <a href="https://hutchdatascience.org/Choosing_Genomics_Tools/index.html" target="_blank" rel="noopener">Online book chapter</a>.</p>
</li>
<li>
<p>Wu S, Furlan S, Mihalas A, Kaya-Okur H, Feroze H, Emerson S, <strong>Zheng Y</strong>, Carson K, Cimino P, Keene C, Holland E, Sarthy J, Gottardo R, Ahmad K, Henikoff S, Patel A. [Single-cell CUTTag analysis of chromatin modifications in differentiation and tumor progression](https://doi. org/10.1038/s41587-021-00865-z). <em><strong>Nature Biotechnology</strong></em>. 2021.</p>
</li>
<li>
<p><strong>Zheng Y</strong>, Ahmad K, Henikoff K. CUTTag Data Processing and Analysis Tutorial. Protocols.io. (2020) <a href="https://www.protocols.io/view/cut-amp-tag-data-processing-and-analysis-tutorial-e6nvw93x7gmk/v1" target="_blank" rel="noopener">https://www.protocols.io/view/cut-amp-tag-data-processing-and-analysis-tutorial-e6nvw93x7gmk/v1</a>. (17,459 views, 4,355 exports, and 239 questions)</p>
</li>
<li>
<p>Liao R+, <strong>Zheng Y+</strong>, Liu X, Zhang Y, Seim G, Tanimura N, Wilson G, Hematti P, Coon J, Fan J, Xu J, Keleş S++ and Bresnick E++. <a href="https://doi.org/10.1016/j.celrep.2020.107832" target="_blank" rel="noopener">Discovering How Heme Controls Genome Function Through Heme-omics</a>. <em><strong>Cell Reports</strong></em>. 2020.</p>
</li>
<li>
<p>Zeng X, Li B, Welch R, Rojo C, <strong>Zheng Y</strong>, Dewey CN, Keleş S. <a href="https://doi.org/10.1371/journal.pcbi.1004491." target="_blank" rel="noopener">Perm-seq: Mapping Protein-DNA Interactions in Segmental Duplication and Highly Repetitive Regions of Genomes with Prior-enhanced Read Mapping</a>. <em><strong>PLoS Computational Biology</strong></em>. 2015.</p>
</li>
</ol>
<h3 id="three-dimensional-chromatin-interaction-and-long-range-gene-regulation">Three-dimensional chromatin interaction and long-range gene regulation</h3>
<ol start="8">
<li>
<p><strong>Zheng Y+</strong>, Shen S+, Keleş S. <a href="https://doi.org/10.1101/2021.03.10.434870" target="_blank" rel="noopener">Normalization and de-noising of single-cell Hi-C data with BandNorm and scVI-3D</a>. Accepted by <em><strong>Genome Biology</strong></em>. 2022. (*: co-first authors)</p>
</li>
<li>
<p>Shen S, <strong>Zheng Y++</strong>, Keleş S++. <a href="https://academic.oup.com/bioinformatics/article-abstract/38/14/3642/6598798?redirectedFrom=fulltext" target="_blank" rel="noopener">scGAD: single-cell gene associating domain scores for exploratory analysis of scHi-C data</a>. <em><strong>Bioinformatics</strong></em>. 2022.  (+: co- corresponding authors)</p>
</li>
<li>
<p>Cheng J, Clayton J, Acemel R, <strong>Zheng Y</strong>, Taylor R, Keleş S, Harley J, Quail E, Gómez-Skarmeta J and Ulgiati D. <a href="https://www.frontiersin.org/articles/10.3389/fimmu.2022.901747/full" target="_blank" rel="noopener">Regulatory architecture of the RCA gene cluster captures an intragenic TAD boundary and enhancer elements in B cells</a>. <em><strong>Frontiers in Immunology, section B Cell Biology</strong></em>. 2022.</p>
</li>
<li>
<p><strong>Zheng Y++</strong>, Zhou P, Keleş S++. <a href="https://www.sciencedirect.com/science/article/abs/pii/S1046202320300852?via%3Dihub" target="_blank" rel="noopener">FreeHi-C Spike-in Simulations for Benchmarking Differential Chromatin Interaction Detection</a>. <em><strong>Methods</strong></em>. 2021.</p>
</li>
<li>
<p>Huang K, Wu Y, Shin J, <strong>Zheng Y</strong>, Siahpirani A, Lin Y, Ni Z, Chen J, You J, Keleş S, Wang D, Roy S, Lu Q. <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1009309" target="_blank" rel="noopener">Transcriptome-wide transmission disequilibrium analysis identifies novel risk genes for autism spectrum disorder</a>. <em><strong>PLOS Genetics</strong></em>. 2021.</p>
</li>
<li>
<p><strong>Zheng Y</strong>, Keleş S. <a href="https://www.nature.com/articles/s41592-019-0624-3" target="_blank" rel="noopener">FreeHi-C simulates high-fidelity Hi-C data for benchmarking and data augmentation</a>. <em><strong>Nature Methods</strong></em>. 2020.</p>
</li>
<li>
<p>The ENCODE Project Consortium, et al. <a href="https://doi.org/10.1038/s41586-020-2493-4" target="_blank" rel="noopener">Expanded Encyclopedias of DNA Elements in the Human and Mouse Genomes</a>. <em><strong>Nature</strong></em>. 2020 .</p>
</li>
<li>
<p>The ENCODE Project Consortium, Snyder, M.P., Gingeras, T.R., Moore, J.E., Weng, Z., Gerstein, M.B., Ren, B., Hardison, R.C., Stamatoyannopoulos, J.A., Graveley, B.R., Feingold, E.A. and Pazin, M.J. <a href="https://doi.org/10.1038/s41586-020-2449-8" target="_blank" rel="noopener">Perspectives on ENCODE</a>. <em><strong>Nature</strong></em>. 2020.</p>
</li>
<li>
<p><strong>Zheng Y</strong>, Ay F, Keleş S. <a href="https://elifesciences.org/articles/38070" target="_blank" rel="noopener">Generative modeling of multi-mapping reads with mHi-C advances analysis of Hi-C studies</a>. <em><strong>eLife</strong></em>. 2019.</p>
</li>
</ol>
<h3 id="single-cell-proteomics">Single-cell Proteomics</h3>
<ol start="17">
<li><strong>Zheng Y+</strong>, Caron D+, Kim J, Jun S, Tian Y, Florian M, Stuart K, Sims P, Gottardo R. ADTnorm: Robust Integration of Single-cell Protein Measurement across CITE-seq Datasets. BioRxiv. Under review of Nature Communications. (2024) <a href="https://www.biorxiv.org/content/10.1101/2022.04.29.489989v2" target="_blank" rel="noopener">https://www.biorxiv.org/content/10.1101/2022.04.29.489989v2</a>.</li>
</ol>
<h3 id="statistical-modeling-and-computational-analysis-of-immunological-and-immunotherapeutic-studies">Statistical Modeling and Computational Analysis of Immunological and Immunotherapeutic Studies</h3>
<ol start="18">
<li>
<p>Fiorenza S, <strong>Zheng Y</strong>, Purushe J, Bock T, Sarthy J, Janssens D, Sheih A, Kimble E, Kirchmeier D, Phi T, Gauthier J, Hirayama A, Riddell S, Wu Q, Gottardo R, Maloney D, Yang J, Henikoff S, Turtle C. Histone marks identify novel transcription factors that parse CAR-T subset-of-origin, clinical potential and expansion. Accepted at <em><strong>Nature Communications</strong></em>. (2024)</p>
</li>
<li>
<p>Germanos AA, Arora S, <strong>Zheng Y</strong>, Goddard ET, Coleman IM, Ku AT, Wilkinson S, Amezquita RA, Zager M, Long A, Yang YC, Bielas J, Gottardo R, Ghajar C, Nelson P, Sowalsky A, Setty M, Hsieh A. <a href="https://elifesciences.org/articles/79076" target="_blank" rel="noopener">Defining cellular population dynamics at single cell resolution during prostate cancer progression</a>. <em><strong>eLife</strong></em>. 2022.</p>
</li>
<li>
<p>Hirayama AV, <strong>Zheng Y</strong>, Dowling MR, Sheih A, Phi TD, Kirchmeier DR, Chucka AW, Gauthier J, Maloney DG, Gottardo R, Turtle CJ. <a href="https://doi.org/10.1182/blood-2021-151571" target="_blank" rel="noopener">Long-Term Follow-up and Single-Cell Multiomics Characteristics of Infusion Products in Patients with Chronic Lymphocytic Leukemia Treated with CD19 CAR-T Cells</a>. <em><strong>Blood</strong></em>. 2021.</p>
</li>
<li>
<p>Vitanza N, Biery M, Myers C, Ferguson E, <strong>Zheng Y</strong>, Girard E, Przystal J, Park G, Noll A, Pakiam F, Winter C, Morris S, Sarthy J, Cole B, Leary S, Crane C, Lieberman N, Mueller S, Nazarian J, Gottardo R, Brusniak M, Mhyre A, Olson J, <a href="https://doi.org/10.1093/neuonc/noaa249" target="_blank" rel="noopener">Optimal therapeutic targeting by HDAC inhibition in biopsy-derived treatment-naïve diffuse midline glioma models.</a> <em><strong>Neuro-Oncology</strong></em>. 2021.</p>
</li>
<li>
<p>Soukup AA, <strong>Zheng Y</strong>, Mehta C, Liu P, Hofmann I, Zhou Y, Zhang J, Choi K, Johnson KD, Keles S, Bresnick EH. <a href="https://www.jci.org/articles/view/122694" target="_blank" rel="noopener">Single-nucleotide human disease mutation inactivates a blood-regenerative GATA2 enhancer</a>. <em><strong>Journal of Clinical Investigation</strong></em>. 2019.</p>
</li>
<li>
<p>Tanimura N, Liao R, Wilson GM, Dent MR, Cao M, Burstyn JN, Hematti P, Liu X, Zhang Y, <strong>Zheng Y</strong>, Keleş S, Xu J, Coon J, Bresnick E. <a href="https://doi.org/10.1016/j.devcel.2018.07.022" target="_blank" rel="noopener">GATA/Heme Multi-omics Reveals a Trace Metal-dependent Cellular Differentiation Mechanism</a>. <em><strong>Developmental Cell</strong></em>. 2018.</p>
</li>
</ol>


  

  
  
  

</div>


    
      </div>
    

    </div>
  </section>

  

  
  
  
  

  

  

  
  
  
  

  

  

  

  
  

  

  
  

  
  
  

  
  
  
  
  

  
  

  

  
  

  
  

  
  <section id="projects" class="home-section wg-portfolio  "  >
   <div class="home-section-bg " >
     
   </div>
    <div class="container">

    
      <div class="row  ">
      
        
          <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
            <h1 class="mb-0">Research Projects</h1>
            
          </div>
        
      
    

      










<div class="col-12 col-lg-8">

  

  

    

    
    
    
    
      
    

    <span class="d-none default-project-filter">*</span>

    
    
    <div class="project-toolbar">
      <div class="project-filters">
        <div class="btn-toolbar ">
          <div class="btn-group flex-wrap">
            
              
              
              
                
              
              <a href="#" data-filter="*" class="btn btn-primary btn-lg active">All</a>
            
              
              
              
              <a href="#" data-filter=".js-id-3D-genomics" class="btn btn-primary btn-lg">3D Genomics</a>
            
              
              
              
              <a href="#" data-filter=".js-id-Immunology" class="btn btn-primary btn-lg">Immunotherapy</a>
            
              
              
              
              <a href="#" data-filter=".js-id-Integration" class="btn btn-primary btn-lg">Multi-omics Integration</a>
            
              
              
              
              <a href="#" data-filter=".js-id-Repetitive-Region" class="btn btn-primary btn-lg">Repetitive Region</a>
            
          </div>
        </div>
      </div>
    </div>
    
  

  <div class="isotope projects-container js-layout-masonry ">

    
    

    
    
      
      
    
    
    

    
    
    
    
    

    
      
        
          <div class="project-card project-item isotope-item js-id-3D-genomics">
        
        











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Jul 31, 2022
  </span>
  

  

  

  
  
  
  

  
  

</div>

  

  
  
  
  
  <a href="/project/3dgenome/" >
    <div class="img-hover-zoom">
      <img src="/project/3dgenome/featured_hu4c5893c39aaf4d6af7556280ba094c8f_242358_808x455_fill_q100_h2_lanczos_smart1_3.webp" height="455" width="808"
           class="article-banner" alt="3D Chromatin Organization" loading="lazy">
    </div>
  </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/project/3dgenome/" >3D Chromatin Organization</a>
  </div>

  
  <a href="/project/3dgenome/"  class="summary-link">
    <div class="article-style">
      <p>Study the three-dimensional chromatin interactions and long-range gene regulation</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  












  






  </div>
  

</div>

      </div>
    
      
        
          <div class="project-card project-item isotope-item js-id-Immunology">
        
        











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Jul 25, 2022
  </span>
  

  

  

  
  
  
  

  
  

</div>

  

  
  
  
  
  <a href="/project/immunotherapy/" >
    <div class="img-hover-zoom">
      <img src="/project/immunotherapy/featured_hu56b5d86735a7a670e9181c6b2067e00f_844663_808x455_fill_q100_h2_lanczos_smart1_3.webp" height="455" width="808"
           class="article-banner" alt="CAR-T cell therapy genomic profiling and clinical association study." loading="lazy">
    </div>
  </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/project/immunotherapy/" >CAR-T cell therapy genomic profiling and clinical association study.</a>
  </div>

  
  <a href="/project/immunotherapy/"  class="summary-link">
    <div class="article-style">
      <p>CAR-T cell therapy cell atlas construction and clinical associated genomic markers detection.</p>
    </div>
  </a>
  

  

</div>

      </div>
    
      
        
          <div class="project-card project-item isotope-item js-id-Integration">
        
        











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Jul 22, 2022
  </span>
  

  

  

  
  
  
  

  
  

</div>

  

  
  
  
  
  <a href="/project/multi-omics/" >
    <div class="img-hover-zoom">
      <img src="/project/multi-omics/featured_hu567d64c75a6634d152911334d7854488_1130301_808x455_fill_q100_h2_lanczos_smart1_3.webp" height="455" width="808"
           class="article-banner" alt="Single-cell Multi-omics Integration and Modeling" loading="lazy">
    </div>
  </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/project/multi-omics/" >Single-cell Multi-omics Integration and Modeling</a>
  </div>

  
  <a href="/project/multi-omics/"  class="summary-link">
    <div class="article-style">
      <p>Statistical modeling and computational analysis of single-cell multi-omics data.</p>
    </div>
  </a>
  

  

</div>

      </div>
    
      
        
          <div class="project-card project-item isotope-item js-id-Repetitive-Region">
        
        











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    Oct 26, 2019
  </span>
  

  

  

  
  
  
  

  
  

</div>

  

  
  
  
  
  <a href="/project/repetitiveregion/" >
    <div class="img-hover-zoom">
      <img src="/project/repetitiveregion/featured_hu6b11dd42e6c9e753137cb68b0b6325f3_166699_808x455_fill_q100_h2_lanczos_smart1_3.webp" height="455" width="808"
           class="article-banner" alt="Repetitive Regions of Genomes" loading="lazy">
    </div>
  </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/project/repetitiveregion/" >Repetitive Regions of Genomes</a>
  </div>

  
  <a href="/project/repetitiveregion/"  class="summary-link">
    <div class="article-style">
      <p>Studies of repetitive regions of genomes from 2D and 3D perspectives</p>
    </div>
  </a>
  

  

</div>

      </div>
    

  </div>
</div>


    
      </div>
    

    </div>
  </section>

  

  
  
  
  

  

  

  
  
  
  

  

  

  

  
  

  

  
  

  
  
  

  
  
  
  
  

  
  

  

  
  

  
  

  
  <section id="software" class="home-section wg-blank  "  >
   <div class="home-section-bg " >
     
   </div>
    <div class="container">

    
      <div class="row  ">
      
        
          <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
            <h1 class="mb-0">Software</h1>
            
          </div>
        
      
    

      



  <div class="col-12 col-lg-8">
    <ul>
<li>
<p><a href="https://github.com/yezhengSTAT/ADTnorm" target="_blank" rel="noopener">ADTnorm</a>: R package for normalization and integration tools for CITE-seq cell surface measurement.</p>
</li>
<li>
<p><a href="https://sshen82.github.io/BandNorm/articles/scGAD-tutorial.html" target="_blank" rel="noopener">scGAD</a>: R package for extracting the three-dimensional chromatin interaction at the unit of genes and facilitate the integration of single-cell 3D genomcis with other single-cell modalities.</p>
</li>
<li>
<p><a href="https://github.com/yezhengSTAT/3DVI" target="_blank" rel="noopener">scVI-3D</a>: Normalization and de-noising of single-cell Hi-C data using deep generative modeling using python pipline.</p>
</li>
<li>
<p><a href="https://github.com/keleslab/BandNorm" target="_blank" rel="noopener">BandNorm</a>: R package for fast band normalization for sing-cell Hi-C data. (Co-developer)</p>
</li>
<li>
<p><a href="https://github.com/yezhengSTAT/FreeHiC/tree/master/SpikeInModule" target="_blank" rel="noopener">FreeHiC Spike-In</a>: FreeHi-C python pipeline with a user/data-driven spike-in module to allow a comprehensive comparison of differential chromatin interaction detection methods where the ground truth differential chromatin interactions are known.</p>
</li>
<li>
<p><a href="https://github.com/yezhengSTAT/FreeHiC" target="_blank" rel="noopener">FreeHiC</a>: Python pipeline using <strong>FR</strong>agment Interactions <strong>E</strong>mpirical <strong>E</strong>stimation method for fast simulation of <strong>Hi-C</strong> and other 3D proximity ligation sequencing data. Major computing parts are accelerated by C.</p>
</li>
<li>
<p><a href="https://github.com/yezhengSTAT/mHiC" target="_blank" rel="noopener">mHiC</a>: Python pipeline of <strong>m</strong>ulti-mapping strategy for <strong>Hi-C</strong> data by probabilistically assigning reads originatedfrom repetitive regions. Major computing parts are accelerated by C.</p>
</li>
<li>
<p><a href="https://github.com/yezhengSTAT/permseq" target="_blank" rel="noopener">permseq</a>: R package for mapping protein-DNA interactions in highly repetitive regions of the genomes with prior-enhanced read mapping.</p>
</li>
<li>
<p><a href="https://github.com/yezhengSTAT/permseqExample" target="_blank" rel="noopener">permseqExample</a>: R package for the permseq package illustration and demo runs. Smaller raw data and demo R scripts are provided for quick runs in order to get to know permseq package.</p>
</li>
</ul>

  </div>



    
      </div>
    

    </div>
  </section>

  

  
  
  
  

  

  

  
  
  
  

  

  

  

  
  

  

  
  

  
  
  

  
  
  
  
  

  
  

  

  
  

  
  

  
  <section id="skills" class="home-section wg-featurette  "  >
   <div class="home-section-bg " >
     
   </div>
    <div class="container">

    

      


<div class="row featurette">
  
  <div class="col-md-12 section-heading">
    <h1>Computing Skills</h1>
    
  </div>
  

  

  
  
  
  
    
  
  <div class="col-12 col-sm-4">
    
    <div class="featurette-icon"><i class="fab fa-r-project"></i></div>
    
    <div class="section-subheading">R</div>
    <p>Daily Usage</p>
  </div>
  
  
  
  
    
  
  <div class="col-12 col-sm-4">
    
    <div class="featurette-icon"><i class="fab fa-linux"></i></div>
    
    <div class="section-subheading">shell</div>
    <p>Daily Usage</p>
  </div>
  
  
  
  
    
  
  <div class="col-12 col-sm-4">
    
    <div class="featurette-icon"><i class="fab fa-python"></i></div>
    
    <div class="section-subheading">Python</div>
    <p>Optional if excel at R</p>
  </div>
  
  
  
  
    
  
  <div class="col-12 col-sm-4">
    
    <div class="featurette-icon"><i class="fab fa-markdown"></i></div>
    
    <div class="section-subheading">Rmarkdown/Jupyter Notebook</div>
    <p>Reproducible Report</p>
  </div>
  
  
  
  
    
  
  <div class="col-12 col-sm-4">
    
    <div class="featurette-icon"><i class="fab fa-git"></i></div>
    
    <div class="section-subheading">git</div>
    <p>Reproducible Research</p>
  </div>
  
  
  
  
    
  
  <div class="col-12 col-sm-4">
    
    <div class="featurette-icon"><i class="fas fa-server"></i></div>
    
    <div class="section-subheading">Grid/Distributed computing systems</div>
    <p>Daily Usage</p>
  </div>
  
</div>


    

    </div>
  </section>

  

  
  
  
  

  

  

  
  
  
  

  

  

  

  
  

  

  
  

  
  
  

  
  
  
  
  

  
  

  

  
  

  
  

  
  <section id="teaching" class="home-section wg-blank  "  >
   <div class="home-section-bg " >
     
   </div>
    <div class="container">

    
      <div class="row  ">
      
        
          <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
            <h1 class="mb-0">Teaching and Mentoring</h1>
            
          </div>
        
      
    

      



  <div class="col-12 col-lg-8">
    <h2 id="teaching">Teaching</h2>
<ul>
<li>
<p><strong>STAT 423/623 - Probability Bioinformatics and Genetics</strong> (Spring 2024 Guest Lecturer at Rice University):</p>
<p>Gave lecture to statistics and biostatistics graduate and undergraduate students about statistical analysis in single-cell genomics.</p>
</li>
<li>
<p><strong>STAT 877 - Statistical Methods for Molecular Biology</strong> (Fall 2020 Guest Lecturer at University of Wisconsin - Madison):</p>
<p>Gave lecture to statistics and biostatistics graduate students about 3D Genomics and Long-range Gene Regulations.</p>
</li>
<li>
<p><strong>AMSI BioInfoSummer</strong> (Winter 2019 Workshop Lecturer at University of Sydney, Australia)</p>
<p>Gave a workshop to the faculty and students attending the AMSI BioInfoSummer conference about basic concepts of 3D Genomics and how to do computational data processing and statistical modeling in a practical manner. Led interactive computational group work to process real Hi-C data using Google Box.</p>
</li>
<li>
<p><strong>STAT 998 - Statistical Consulting</strong> (Fall 2019 Guest Lecturer at University of Wisconsin - Madison):</p>
<p>Lead lectures to discuss real-world consulting problem with statistics graduate students utilizing the traditional and modern statistical tools.</p>
</li>
<li>
<p><strong>STAT 877 - Statistical Methods for Molecular Biology</strong> (Spring 2019 Guest Lecturer at University of Wisconsin - Madison):</p>
<p>Gave lecture to statistics and biostatistics graduate students about 3D Genomics and Long-range Gene Regulations.</p>
</li>
<li>
<p><strong>2017-2018 Single-cell Technologies Journal Club</strong> (Organizer and Instructor at University of Wisconsin - Madison):</p>
<p>Gave lectures about single-cell related research topics, such as scRNA-seq, scATAC-seq and scHi-C, to graduate students and post-docs from statistics background, and led paper review discussions.</p>
</li>
<li>
<p><strong>2017-2018 Three-dimensional Chromatin Interactions Journal Club</strong> (Organizer and Instructor at University of Wisconsin - Madison):</p>
<p>Gave lectures about 3D chromatin architecture related research topics to graduate students and post-docs from statistics background, and led paper review discussions.</p>
</li>
<li>
<p><strong>STAT301 - Introduction to Statistical Methods</strong> (Fall 2014 Guest Lecturer for Discussion Sections at University of Wisconsin - Madison):</p>
<p>Led undergraduate students discussions for solving hypothesis testing and statistical estimation problems.</p>
</li>
</ul>
<h2 id="mentoring">Mentoring</h2>
<ul>
<li>
<p><strong>Long Nguyen, Bioinformatics Analyst I at Fred Hutchinson Cancer Center, currently Master student at University of Michigan</strong> (Feb. 2022 to July 2024):</p>
<p>Single-cell transcriptomics and proteomics integrative analysis for cell atlas construction of CAR-T cell therapy CITE-seq data and association with gene and protein markers with clinical responses.</p>
</li>
<li>
<p><strong>Siqi Shen, Ph.D. Candidate at UW-Madison</strong> (June 2020 to Dec. 2023):</p>
<p>Co-mentor with Dr. Sunduz Keles on single-cell 3D chromatin organization normalization and integrative analysis with single-cell transcriptomics and epigenomics.</p>
</li>
<li>
<p><strong>Fanding Zhou, VISP student at UW-Madison, currently Ph.D. student at UC Berkeley</strong> (June 2020 to  Sep. 2021):</p>
<p>Co-mentor with Dr. Sunduz Keles on constructing tree-based statistical models for the false discovery rate control of 3D chromatin organization differential detection.</p>
</li>
<li>
<p><strong>Olivia Rae Steidl, Summer Undergraduate Student at University of Wisconsin - Madison, currently Ph.D. student at University of Wisconsin - Madison</strong> (Summer 2019):</p>
<p>Co-mentor with Dr. Sunduz Keles on investigation of the poly(UG) tails at the end of RNAs and its function in human using eCLIP-seq data.</p>
</li>
</ul>

  </div>



    
      </div>
    

    </div>
  </section>

  

  
  
  
  

  

  

  
  
  
  

  

  

  

  
  

  

  
  

  
  
  

  
  
  
  
  

  
  

  

  
  

  
  

  
  <section id="activity" class="home-section wg-blank  "  >
   <div class="home-section-bg " >
     
   </div>
    <div class="container">

    
      <div class="row  ">
      
        
          <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
            <h1 class="mb-0">Professional Activity</h1>
            
          </div>
        
      
    

      



  <div class="col-12 col-lg-8">
    <h2 id="journal-reviewer">Journal Reviewer</h2>
<ul>
<li>
<p>Genome Medicine</p>
</li>
<li>
<p>Science Advances</p>
</li>
<li>
<p>Nature Biotechnology</p>
</li>
<li>
<p>Briefings in Bioinformatics</p>
</li>
<li>
<p>Scientific Report</p>
</li>
<li>
<p>eLife</p>
</li>
<li>
<p>Bioinformatics</p>
</li>
<li>
<p>PLOS Computational Biology</p>
</li>
<li>
<p>BMC Bioinformatics</p>
</li>
<li>
<p>Life Science Alliance</p>
</li>
<li>
<p>Annals of Applied Statistics</p>
</li>
<li>
<p>Computational and Structural Biotechnology</p>
</li>
</ul>
<h2 id="committee">Committee</h2>
<ul>
<li>
<p>Grant Review Committee: 2024 Fred Hutchison Cancer Center TDS IRC Postdoctoral Fellowship and TDS IRC Pilot Award</p>
</li>
<li>
<p>Program Committee: 2024 Regulatory and Systems Genomics Conference with DREAM Challenges (RSGDREAM2024)</p>
</li>
</ul>

  </div>



    
      </div>
    

    </div>
  </section>

  

  
  
  
  

  

  

  
  
  
  

  

  

  

  
  

  

  
  

  
  
  

  
  
  
  
  

  
  

  

  
  

  
  

  
  <section id="posts" class="home-section wg-pages  "  >
   <div class="home-section-bg " >
     
   </div>
    <div class="container">

    
      <div class="row  ">
      
        
          <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
            <h1 class="mb-0">Blogs</h1>
            
          </div>
        
      
    

      









  









  
  
  
  
  



















  




<div class="col-12 col-lg-8">

  

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/post/cuttag/" >CUT&amp;Tag Data Processing and Analysis Tutorial</a>
    </div>

    
    <a href="/post/cuttag/"  class="summary-link">
      <div class="article-style">
        This tutorial is designed for processing and analyzing CUT&amp;Tag data following the Benchtop CUT&amp;Tag V.3 protocol.  The illustration data used in this tutorial is the profiling of histone modifications in the human lymphoma K562 cell line, but the tutorial is generally applicable to any chromatin protein, including transcription factors, RNA polymerase II, and epitope-tagged proteins. For reproducible analysis, this tutorial is also available on GitHub at <a href="https://yezhengstat.github.io/CUTTag_tutorial/" target="_blank" rel="noopener">https://yezhengstat.github.io/CUTTag_tutorial/</a>.
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      Ye Zheng</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Aug 12, 2020
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    1 min read
  </span>
  

  
  
  
  

  
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
    
    <a href="/post/cuttag/" >
      <img src="/post/cuttag/featured_huf21eab75994bf75d727a82005b169e9b_2639680_150x0_resize_q100_h2_lanczos_3.webp" height="150" width="150"
           alt="CUT&amp;Tag Data Processing and Analysis Tutorial" loading="lazy">
    </a>
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/post/3dreview/" >Three-dimensional genome architecture and disease variants</a>
    </div>

    
    <a href="/post/3dreview/"  class="summary-link">
      <div class="article-style">
        Three-dimensional genome architecture and disease variants.
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      Ye Zheng</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Jan 19, 2019
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    4 min read
  </span>
  

  
  
  
  

  
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/post/cython/" >Cython - Linking C/C&#43;&#43; with Python</a>
    </div>

    
    <a href="/post/cython/"  class="summary-link">
      <div class="article-style">
        Cython - Linking C/C++ with Python and Accelerate Computation Considerably.
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      Ye Zheng</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
          Last updated on
      
    
    Oct 13, 2018
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    2 min read
  </span>
  

  
  
  
  

  
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/post/ggplot2/" >ggplot2 Summary and Color Recommendation</a>
    </div>

    
    <a href="/post/ggplot2/"  class="summary-link">
      <div class="article-style">
        ggplot2 Summary and Color Recommendation for Clean and Pretty Visualization.
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      Ye Zheng</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
          Last updated on
      
    
    Oct 22, 2018
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    6 min read
  </span>
  

  
  
  
  

  
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/post/sra/" >Download data from GEO</a>
    </div>

    
    <a href="/post/sra/"  class="summary-link">
      <div class="article-style">
        Download data from GEO by linux command lines.
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      Ye Zheng</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
          Last updated on
      
    
    Oct 23, 2018
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    3 min read
  </span>
  

  
  
  
  

  
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/post/bash/" >Practical Shell Commands to Manipulate Genome Data</a>
    </div>

    
    <a href="/post/bash/"  class="summary-link">
      <div class="article-style">
        Practical Shell Commands to Manipulate Genome Data in a Fast and Clean fashion.
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      Ye Zheng</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
          Last updated on
      
    
    Oct 23, 2018
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    3 min read
  </span>
  

  
  
  
  

  
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
  </div>
</div>

  
    













  


<div class="media stream-item view-compact">
  <div class="media-body">

    <div class="section-subheading article-title mb-0 mt-0">
      <a href="/post/r/" >Using R to output strings as clickable links in table</a>
    </div>

    
    <a href="/post/r/"  class="summary-link">
      <div class="article-style">
        outputString &lt;- &lsquo;=HYPERLINK(&ldquo;yourURL&rdquo;)&rsquo;
      </div>
    </a>
    

    <div class="stream-meta article-metadata">

      

      
        


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      Ye Zheng</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
          Last updated on
      
    
    Oct 23, 2018
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    1 min read
  </span>
  

  
  
  
  

  
  

</div>

      
    </div>

    

  </div>
  <div class="ml-3">
    
    
  </div>
</div>

  

  
  
  

</div>


    
      </div>
    

    </div>
  </section>

  

  
  
  
  

  

  

  
  
  
  

  

  

  

  
  

  

  
  

  
  
  

  
  
  
  
  

  
  

  

  
  

  
  

  
  <section id="contact" class="home-section wg-contact  "  >
   <div class="home-section-bg " >
     
   </div>
    <div class="container">

    
      <div class="row  ">
      
        
          <div class="section-heading col-12 col-lg-4 mb-3 mb-lg-0 d-flex flex-column align-items-center align-items-lg-start">
            <h1 class="mb-0">Contact</h1>
            
          </div>
        
      
    

      


















<div class="col-12 col-lg-8">
  

  

  <ul class="fa-ul">

    

    

    
    

    

    

    

    
    
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <i class="fa-li fas fa-envelope fa-2x" aria-hidden="true"></i>
      <a href="/yzheng23@fredhutch.org" >yzheng two three @ fredhutch.org</a>
    </li>
    
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <i class="fa-li fas fa-map-location fa-2x" aria-hidden="true"></i>
      <a href="/" >1100 Fairview Ave N, Seattle, WA</a>
    </li>
    

  </ul>

  

</div>


    
      </div>
    

    </div>
  </section>







  </div>

  <div class="page-footer">
    
    
    <div class="container">
      <footer class="site-footer">

  



  

  

  

  
  






  
  
  

  
  
    
  
  
    
  

  

  
  <p class="powered-by copyright-license-text">
    © 2024 Ye Zheng. This work is licensed under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0" rel="noopener noreferrer" target="_blank">CC BY NC ND 4.0</a>
  </p>
  

  <p class="powered-by footer-license-icons">
    <a href="https://creativecommons.org/licenses/by-nc-nd/4.0" rel="noopener noreferrer" target="_blank" aria-label="Creative Commons">
      <i class="fab fa-creative-commons fa-2x" aria-hidden="true"></i>
      <i class="fab fa-creative-commons-by fa-2x" aria-hidden="true"></i>
      
        <i class="fab fa-creative-commons-nc fa-2x" aria-hidden="true"></i>
      
      
        <i class="fab fa-creative-commons-nd fa-2x" aria-hidden="true"></i>
      
    </a>
  </p>




  <p class="powered-by">
    
    
    
      
      
      
      
      
      
      Published with <a href="https://wowchemy.com/?utm_campaign=poweredby" target="_blank" rel="noopener">Wowchemy</a> — the free, <a href="https://github.com/wowchemy/wowchemy-hugo-themes" target="_blank" rel="noopener">open source</a> website builder that empowers creators.
    
  </p>
</footer>

    </div>
    
  </div>

      

    
    <script src="/js/vendor-bundle.min.46271ef31da3f018e9cd1b59300aa265.js"></script>

    
    
    
      
        <script src="https://cdn.jsdelivr.net/gh/desandro/imagesloaded@v4.1.4/imagesloaded.pkgd.min.js" integrity="sha512-S5PZ9GxJZO16tT9r3WJp/Safn31eu8uWrzglMahDT4dsmgqWonRY9grk3j+3tfuPr9WJNsfooOR7Gi7HL5W2jw==" crossorigin="anonymous"></script>
        <script src="https://cdn.jsdelivr.net/gh/metafizzy/isotope@v3.0.6/dist/isotope.pkgd.min.js" integrity="sha512-Zq2BOxyhvnRFXu0+WE6ojpZLOU2jdnqbrM1hmVdGzyeCa1DgM3X5Q4A/Is9xA1IkbUeDd7755dNNI/PzSf2Pew==" crossorigin="anonymous"></script>
      

      
      

      

      
        
        <script src="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.2.1/build/highlight.min.js" integrity="sha512-Ypjm0o7jOxAd4hpdoppSEN0TQOC19UtPAqD+4s5AlXmUvbmmS/YMxYqAqarQYyxTnB6/rqip9qcxlNB/3U9Wdg==" crossorigin="anonymous"></script>
        
        
        <script src="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.2.1/build/languages/r.min.js" crossorigin="anonymous"></script>
        
        <script src="https://cdn.jsdelivr.net/gh/highlightjs/cdn-release@10.2.1/build/languages/latex.min.js" crossorigin="anonymous"></script>
        
      

    

    
    
    

    
    

    
    
    
      
      <script id="search-hit-fuse-template" type="text/x-template">
        <div class="search-hit" id="summary-{{key}}">
          <div class="search-hit-content">
            <div class="search-hit-name">
              <a href="{{relpermalink}}">{{title}}</a>
              <div class="article-metadata search-hit-type">{{type}}</div>
              <p class="search-hit-description">{{snippet}}</p>
            </div>
          </div>
        </div>
      </script>
      
        <script src="https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js" integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin="anonymous"></script>
        <script src="https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js" integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin="anonymous"></script>
      
    

    
    

    
    
    
    

    
    
      
      
      
      
      
      
      
    

    

    
    
    
    <script id="page-data" type="application/json">{"use_headroom":false}</script>

    
    
    
    
    
    
    
    
    
      
      
    
    
    <script src="/en/js/wowchemy.min.3239609ca7abe02ed52d76ffcb22af1d.js"></script>

    
    
    
    
    
    
      
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

      <script src="/js/wowchemy-publication.68f8d7090562ca65fc6d3cb3f8f2d2cb.js" type="module"></script>






</body>
</html>
 -->
